pkg/container/vector/vector.go

```diff
@@ -811,10 +811,26 @@ func (v *Vector) UnmarshalBinaryWithCopy(data []byte, mp *mpool.MPool) error {
 func (v *Vector) ToConst() {
 	if v.nsp.Contains(0) {
 		v.data = v.data[:0]
+		v.capacity = 0
```

Comment:
capacity is always "cap(v.data) / v.typ.TypeSize()" and only be changed when ResetWithType or extend the capacity.